### PR TITLE
Implement word histogram

### LINKED ISSUE #9: Implement distributed word histogram
Use any MapReduce framework of your choice to implement word histogram.  Note that in Python there's collections.Counter.

### PR DESCRIPTION
**Pull request by Huy-Ngo**
Resolve #9

### CODE REVIEW

File: WordCount/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+grpcio==1.35.0
+grpcio-tools==1.35.0
+mrjob==0.7.4
+protobuf==3.14.0
+PyYAML==5.4.1
+six==1.15.0
```
Comment (McSinyx) [inline WordCount/requirements.txt]: Why so many indirect dependencies, I thought mrjob is enough?

File: WordCount/wc.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+
+class WordCount(MRJob):
+
+    def mapper(self, _, line):
+        for word in word_regex.findall(line):
+            yield (word.lower(), 1)
+
+    def combiner(self, word, counts):
+        yield (word, sum(counts))
```
Comment (McSinyx) [inline WordCount/wc.py]: This one can be safely dropped.

File: WordCount/requirements.txt (hunk 1 above)
Comment (Huy-Ngo) [reply WordCount/requirements.txt]: I did `pip freeze >` so they're there lol.

File: WordCount/wc.py (hunk 2 above)
Comment (Huy-Ngo) [reply WordCount/wc.py]: Oh I thought I pushed but I didn't

File: WordCount/requirements.txt (hunk 1 above)
Comment (Huy-Ngo) [reply WordCount/requirements.txt]: Oh it's dependent on grpc.

**Review by McSinyx — APPROVED** [review]
Good job!

**Pull request merged**